hpat/hiframes/hiframes_typed.py

```diff
@@ -356,23 +356,14 @@ def _run_getattr(self, assign, rhs):
             if rhs.attr in hpat.hiframes.pd_timestamp_ext.timedelta_fields:
                 return self._run_Timedelta_field(assign, assign.target, rhs)
 
-        if isinstance(rhs_type, SeriesType) and rhs.attr in ('size', 'shape'):
+        if isinstance(rhs_type, SeriesType) and rhs.attr in ('size'):
```

Comment:
`rhs.attr in ('size')` checks sub-string in string `'size'`, because `('size')` is a string, not a tuple, e.g. `('size',)`. It's not entirely correct. In case of `rhs.attr` is part of `'size'` we get strange behavior. I propose to use condition `rhs.attr == 'size'` here.

Reply:
Fixed, thank you.